src/overlays/actors/ovl_Eff_Dust/z_eff_dust.c

```diff
@@ -36,7 +36,7 @@ const ActorInit Eff_Dust_InitVars = {
     (ActorFunc)EffDust_Draw,
 };
 
-static Gfx D_8099EB60[] = {
+static Gfx sEndDL[] = {
```

Comment:
```suggestion
static Gfx sEmptyDL[] = {
```

like in https://github.com/zeldaret/oot/blob/master/src/code/z_rcp.c#L846